File: 151001/Kokarev/rest1/src/main/java/by/bsuir/romankokarev/impl/bean/News.java
package by.bsuir.romankokarev.impl.bean;

import lombok.Getter;
import lombok.Setter;

import java.sql.Date;

@Setter
@Getter
public class News {
    private long id;
    private long userId;
    private String title;
    private String content;
    private Date created;
    private Date modified;
}

File: 151004/Bosko/lab1/lab1/Models/News.cs
namespace lab1.Models
{
    public class News
    {
        public int Id { get; set; }

        public int? UserId { get; set; }

        public string? Title { get; set; }

        public string? Content { get; set; }

        public string? Created {  get; set; }

        public string? Modified {  get; set; }

        public virtual User? User { get; set; }

        public virtual ICollection<Note> Notes { get; set; } = new List<Note>();
    }
}
